src/Microsoft.AspNetCore.Authentication.AzureAd/AzureExtensions.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Authentication.AzureAd;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Options.Infrastructure;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class AzureExtensions
```

Comment:
These methods cover the web app case, but we need to also cover the Web API case where we setup JWT bearer authentication.